lib/screens/dashboard.dart

```diff
@@ -802,8 +802,172 @@ class CustomSliverDelegate extends SliverPersistentHeaderDelegate {
                 ),
               ],
             ),
-          );
-        },
+
```

Comment:
@daemon29 , I think the deleted lines 805 & 806 need to be there, and the added lines 805-970 need not be there. This is because the lines 805-970 are already included in lines 503-651

Reply:
@daemon29, Just fixed this problem!